extension/popup.js

```diff
@@ -93,6 +93,7 @@ const quickConvertShortcut = document.getElementById('quickConvertShortcut');
 const contentScopeRadios = document.querySelectorAll('input[name="contentScope"]');
 const preserveTablesCheckbox = document.getElementById('preserveTables');
 const includeImagesCheckbox = document.getElementById('includeImages');
+const includeTitleCheckbox = document.getElementById('includeTitle');
```

Comment:
The way you've consistently implemented the new `includeTitle` setting across all three files (popup.html, popup.js, content.js) follows the established pattern well. I appreciate the consistency